pkg/kubernetes/types.go

```diff
@@ -80,6 +80,9 @@ type Controller interface {
 	// ListMonitoredNamespaces returns the namespaces monitored by the mesh
 	ListMonitoredNamespaces() ([]string, error)
 
+	// GetNamespace returns k8s namespace present in cache
```

Comment:
When I read this comment I wondered - would this namespace be from the list of monitored namespaces? OR would it be any namespace within that k8s cluster.
I also wonder what would happen if the namespace does not exist (what if I have a typo in the string passed).

Do you think these should be documented here for clarity?

Reply:
addressing the comment/doc on a subsequent patch:
I want to try it out again, because I recall that when I was testing the test I added, I could get a namespace that was not tagged with the label. But could still be `testclient` shenanigans maybe.

Reply:
edit: or maybe was just me